### ISSUE TITLE
menu spaz

### ISSUE THREAD

**Issue by mecs13** [mecs13/website_text#60]
- hover over horizontal bar of the menu
- menu starts bugging out, glitching spazzing
- should BE smooth

firefox
macosx

<img width="488" alt="screen shot 2018-02-24 at 12 38 39 pm" src="https://user-images.githubusercontent.com/25008118/36633236-b2609aea-195f-11e8-9867-fa964cf76866.png">

**Comment by tbinetruy** [comment]
Absolutely. This is a very hard bug to handle.  It could involve major refactoring of menu component. Classifying it as low priority

**Comment by tbinetruy** [comment]
I can either leave it like that until we have time to fix or remove dynamics in the menu entirely and go back to static menu. Much uglier and not fun, but won't be glitchy.
